cmd/ctl/go.mod

```diff
@@ -12,45 +12,45 @@ go 1.20
 // or a branch name (master).
 
 require (
-	github.com/cert-manager/cert-manager v1.13.0-alpha.0.0.20230801130528-b93ec2f8242b
+	github.com/cert-manager/cert-manager v1.13.0-alpha.0.0.20230825134310-63cf4e0b1c56
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.2
-	golang.org/x/crypto v0.6.0
-	helm.sh/helm/v3 v3.12.0
-	k8s.io/api v0.27.4
-	k8s.io/apiextensions-apiserver v0.27.4
-	k8s.io/apimachinery v0.27.4
-	k8s.io/cli-runtime v0.27.4
-	k8s.io/client-go v0.27.4
-	k8s.io/component-base v0.27.4
-	k8s.io/kubectl v0.27.4
-	k8s.io/utils v0.0.0-20230711102312-30195339c3c7
-	sigs.k8s.io/controller-runtime v0.15.0
+	github.com/stretchr/testify v1.8.4
+	golang.org/x/crypto v0.12.0
+	helm.sh/helm/v3 v3.12.3
+	k8s.io/api v0.28.1
+	k8s.io/apiextensions-apiserver v0.28.1
+	k8s.io/apimachinery v0.28.1
+	k8s.io/cli-runtime v0.28.1
+	k8s.io/client-go v0.28.1
+	k8s.io/component-base v0.28.1
+	k8s.io/kubectl v0.28.1
+	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
+	sigs.k8s.io/controller-runtime v0.16.0
```

Comment:
How come these didn't get upgraded in https://github.com/cert-manager/cert-manager/pull/6287 ?

Is it because it MUST be done in two steps? everything-else *then* ctl. Or was it just an oversight?

Reply:
I'm not sure, go-mod-upgrade only showed this upgrade now & did not show it in the other PR (as far as I remember).